src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/model/SwitchPairs.groovy

```diff
@@ -42,4 +51,9 @@ class SwitchPairs {
         assumeFalse(switchPairs.isEmpty(), "No suiting switch pair found")
         return switchPairs.first()
     }
+
+    SwitchPair pairWithDesiredSwitch(Switch sw) {
+        assumeFalse(switchPairs.isEmpty(), "No suiting switch pair found")
```

Comment:
You don't need these assumptions.
Refactor method to have signature like  `SwitchPairs includeSwitch(Switch sw)`  and use 'random()' method from this class in test.

Reply:
Done.